defaults/main.yml

```diff
@@ -4,6 +4,9 @@ zookeeper_playbook_version: "0.9.2"
 zookeeper_version: 3.4.6
 zookeeper_url: http://www.us.apache.org/dist/zookeeper/zookeeper-{{zookeeper_version}}/zookeeper-{{zookeeper_version}}.tar.gz
 
+# Flag that selects if systemd or upstart will be used for the init service:
+# Note: by default Ubuntu 15.04 and later use systemd (but support switch to upstart)
+zookeeper_debian_systemd_enabled: "{{ ansible_distribution_version|version_compare(15.04, '>=') }}"
```

Comment:
will do!
ps: adding more supported Ubuntu versions (not sure about the existing old(est) ones in that list !)
```
  - name: Ubuntu
    versions:
      - all
...
      - utopic
      - vivid
      - wily
      - xenial
```